src/appimagetool.c

```diff
@@ -61,6 +61,8 @@ extern char runtime[];
 extern unsigned int runtime_len;
 #endif
 
+extern char **environ;
```

Comment:
You shouldn't define this on your own; instead, you could import it from an included header. However, you don't need it at all (see below).